apps/dashboard/src/components/dashboard-layout.tsx

```diff
@@ -9,16 +9,20 @@ import { HeaderNavigation } from '@/components/header-navigation/header-navigati
 export const DashboardLayout = ({
   children,
   headerStartItems,
+  showSideNavigation = true,
+  showBridgeUrl = true,
 }: {
   children: ReactNode;
   headerStartItems?: ReactNode;
+  showSideNavigation?: boolean;
+  showBridgeUrl?: boolean;
 }) => {
   return (
     <IntercomProvider appId={INTERCOM_APP_ID}>
       <div className="relative flex h-full w-full">
-        <SideNavigation />
+        {showSideNavigation && <SideNavigation />}
```

Comment:
for the Vercel integration route we hide side nav and local studio header button